conf/schemas/streamquery.json

```diff
@@ -0,0 +1,17 @@
+{
+  "streamquery:version1": {
```

Comment:
Don't we all rename `streamquery` to `scheduled_queries`, including file name.

Reply:
In this specific case, I would challenge this. The the schema convention namespaces a `product:type`. `scheduled_queries` is not the name of a product. 

We could do it like:
* `streamalert:scheduled_queries:version1`
* `streamquery:version1`

StreamQuery is the *pet name* of the product, and is not something that we should just delete. It's similar to `LookupTables` being the pet name of the other product